### PR TITLE
Fixing Tier4 sync_plan tests

### LINKED ISSUE #3778: Fix syncplan tests for tier4

### PR DESCRIPTION
**Pull request by oshtaier**
Closes #3778
Tested both locally and using standalone job with docker containers(208):
```
tests/foreman/ui/test_syncplan.py::SyncPlanTestCase::test_negative_create_with_invalid_name PASSED
tests/foreman/ui/test_syncplan.py::SyncPlanTestCase::test_negative_create_with_same_name PASSED
tests/foreman/ui/test_syncplan.py::SyncPlanTestCase::test_positive_create_with_description PASSED
tests/foreman/ui/test_syncplan.py::SyncPlanTestCase::test_positive_create_with_name PASSED
tests/foreman/ui/test_syncplan.py::SyncPlanTestCase::test_positive_create_with_start_date PASSED
tests/foreman/ui/test_syncplan.py::SyncPlanTestCase::test_positive_create_with_start_time PASSED
tests/foreman/ui/test_syncplan.py::SyncPlanTestCase::test_positive_create_with_sync_interval PASSED
tests/foreman/ui/test_syncplan.py::SyncPlanTestCase::test_positive_delete PASSED
tests/foreman/ui/test_syncplan.py::SyncPlanTestCase::test_positive_synchronize_custom_product_future_sync_date PASSED
tests/foreman/ui/test_syncplan.py::SyncPlanTestCase::test_positive_synchronize_custom_products_future_sync_date PASSED
tests/foreman/ui/test_syncplan.py::SyncPlanTestCase::test_positive_synchronize_rh_product_future_sync_date PASSED
tests/foreman/ui/test_syncplan.py::SyncPlanTestCase::test_positive_update_and_disassociate_product PASSED
tests/foreman/ui/test_syncplan.py::SyncPlanTestCase::test_positive_update_interval PASSED
tests/foreman/ui/test_syncplan.py::SyncPlanTestCase::test_positive_update_name PASSED
tests/foreman/ui/test_syncplan.py::SyncPlanTestCase::test_positive_update_product PASSED
```

### CODE REVIEW

File: tests/foreman/ui/test_syncplan.py [hunk 1]
```diff
@@ -589,8 +612,9 @@ def test_positive_synchronize_custom_products_future_sync_date(self):
             # Associate sync plan with products
             self.syncplan.update(
                 plan_name, add_products=[product.name for product in products])
-            # Wait half of expected time
-            sleep(delay/2)
+            # Wait third part of expected time, because it will take a while to
+            # verify each product and repository
+            sleep(delay/3)
```
Comment (rochacbruno) [inline tests/foreman/ui/test_syncplan.py]: missing space between operator `sleep(delay / 3)`

File: tests/foreman/ui/test_syncplan.py [hunk 2]
```diff
@@ -600,7 +624,7 @@ def test_positive_synchronize_custom_products_future_sync_date(self):
                     after_sync=False,
                 )
             # Wait the rest of expected time
-            sleep(delay/2)
+            sleep(delay*2/3)
```
Comment (rochacbruno) [inline tests/foreman/ui/test_syncplan.py]: `sleep(delay * 2 / 3)` ?

**Comment by rochacbruno** [comment]
ACK pending non-blocker styling nitpick

**Comment by sghai** [comment]
ACK

**Pull request merged**